student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(_id, age, classroom, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly remove `_id` from the list of parameters here as it auto-generated in the Person class so we don't need to pass it in.

- [ ] In the `super` parameters kindly re-arrange them, `age` should come before `name` based on how the Person constructor is set up.

  ```
  super(age, name, parent_permission: parent_permission)
  ```